web.md

```diff
@@ -0,0 +1,37 @@
+# Web
+
+**Prerequisite knowledge**: 
+
+## Intro
+Web - short for World Wide Web.
+
+No prerequisite knowledge required. If you see this document, you are probably aware of what computers are and that they can communicate with each other via global network of computer networks, called the Internet. If so, you're all set to begin your journey with the web.
+
+## Classification / acceptance questions
+### Apprentice:
+- I can tell what World Wide Web is.
+- I can tell the difference betweend web and the internet.
+- I know what a web browser is.
+- I know what a web server is.
```

Comment:
We could also mention about supported media types.

Reply:
🤔Uhm, you mean media supported by HTML5?  Specific formats like mp3? Response mimetypes supported by browsers? I think, I need an additional hint on that one ✋🏻

Reply:
I was thinking about listing "images, video, and audio". This gives you an overall view on web capabilities but if you need more info you can dive into details like formats etc.